Deeploy/Targets/Generic/TypeCheckers.py

```
@@ -598,6 +598,21 @@ def _inferSignedness(self, inputs: List[VariableBuffer],
         return [True]
 
 
+class AdamChecker(SignPropTypeChecker):
+
+    def __init__(self, input_types: Sequence[Type[Pointer]], output_types: Sequence[Type[Pointer]]):
+        super().__init__(input_types, output_types)
+
+    def _inferNumLevels(self, inputs: List[VariableBuffer],
```

Ruff (0.15.2)

[warning] Unused method argument: `inputs`

(ARG002)